### PR TITLE
Render the html properly returned by server in the notification dropdown

### LINKED ISSUE #744: Render the html sent by server properly in the notification dropdown.
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
notificationhtml

**Expected behavior:**
notificationhtmlexpected

### PR DESCRIPTION
**Pull request by sumedh123**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Fixes #744

### CODE REVIEW

File: app/templates/components/notification-dropdown.hbs [hunk 1]
```diff
@@ -23,7 +23,7 @@
             {{/link-to}}
           </div>
           <div class="content weight-600">
-            {{notification.message}}
+            {{html-safe notification.message}}
```
Comment (niranjan94) [inline app/templates/components/notification-dropdown.hbs]: Why do you need a new helper for this when `sanitize` is already there ?

Comment (sumedh123) [inline app/templates/components/notification-dropdown.hbs]: @niranjan94 Sorry I missed that. Will remove the new one.

**Comment by niranjan94** [comment]
Deployment link: https://pr-745-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@geekyd @CosmicCoder96 @harshitagupta30 @niranjan94 Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-745-open-event-frontend-fossasia.surge.sh

**Comment by sumedh123** [comment]
@geekyd Please review.

**Comment by niranjan94** [comment]
Deployment link: https://pr-745-open-event-frontend-fossasia.surge.sh

**Pull request merged**